core/src/main/java/org/mskcc/cbio/portal/util/GenePanelUtil.java

```diff
@@ -137,6 +146,20 @@ public Pair(Set<CanonicalGene> add, Set<CanonicalGene> remove) {
             this.add = add;
             this.remove = remove;
         }
+
+        @Override
+        public boolean equals(Object o) {
```

Comment:
needed to implement equals so assertEquals within the unit tests won't throw errors